reference/docs-conceptual/learn/shell/using-predictors.md

```diff
@@ -0,0 +1,149 @@
+---
+description: This article describes the features and usage of Predictive IntelliSense in PSReadLine.
+no-loc: [<kbd>Tab</kbd>, <kbd>F2</kbd>, <kbd>RightArrow</kdb>, <kbd>Enter</kbd>]
+title: Using predictors is PSReadLine
+ms.date: 7/14/2022
+---
+# Using predictors is PSReadLine
+
+**PSReadLine** 2.1.0 introduced the **Predictive IntelliSense** feature. Predictive IntelliSense
+provides suggestions for full commands based on items from your **PSReadLine** history.
+**PSReadLine** 2.2.2 extends the power of Predictive IntelliSense by adding support for plug-in
+modules that use advanced logic to provide suggestions for full commands. The latest version,
+**PSReadLine** 2.2.6, enables predictions by default.
+
+## Using Predictive IntelliSense
+
+When Predictive IntelliSense is enabled, the prediction suggestion appears as colored text following
+the user's cursor. The suggestions from Predictive IntelliSense help new and experienced users of
+PowerShell discover, edit, and execute full commands based on matching predictions. Suggestions can
+come from the user's history and additional domain specific plugins.
```

Comment:
We should clarify that users history in this case is different than `Get-History` and is persistent when opening new sessions

Reply:
Done